### PR TITLE
CLOUDP-164740 Add e2e tests for "backup schedule"

### PR DESCRIPTION
**Pull request by fmenezes**
## Proposed changes

Add e2e tests for "backup schedule"

_Jira ticket:_ CLOUDP-164740

## Checklist

- [ ] I have signed the [MongoDB CLA](https://www.mongodb.com/legal/contributor-agreement)
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added any necessary documentation in document requirements section listed in [CONTRIBUTING.md](https://github.com/mongodb/mongodb-atlas-cli/blob/master/CONTRIBUTING.md) (if appropriate)
- [ ] I have addressed the @mongodb/docs-cloud-team comments (if appropriate)
- [ ] I have updated [test/README.md](https://github.com/mongodb/mongodb-atlas-cli/blob/master/test/README.md) (if an e2e test has been added)
- [ ] I have run `make fmt` and formatted my code

## Further comments

Adds e2e tests for:
- `atlas backup schedule describe`
- `atlas backup schedule update`
- `atlas backup schedule delete`

### CODE REVIEW

**Review by blva — DISMISSED** [review]
LGTM

File: test/e2e/atlas/backup_schedule_test.go [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+	"os/exec"
+	"testing"
+
+	"github.com/mongodb/mongodb-atlas-cli/test/e2e"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	atlas "go.mongodb.org/atlas/mongodbatlas"
+)
+
+func TestSchedules(t *testing.T) {
```
Comment (gssbzn) [inline test/e2e/atlas/backup_schedule_test.go:31]: question, any reason we are not using the test generator to create projects and clusters? right now this approach risk hitting the clusters limit per project we've seen in the pass

Comment (fmenezes) [reply test/e2e/atlas/backup_schedule_test.go:31]: Alright, I'll take a look

**Review by gssbzn — APPROVED** [review]
thanks

**Review by gssbzn — APPROVED** [review]
LGTM

**Pull request merged**